package-lock.json

```diff
@@ -1816,7 +1816,6 @@
         "anymatch": "^2.0.0",
         "async-each": "^1.0.1",
         "braces": "^2.3.2",
-        "fsevents": "^1.2.7",
```

Comment:
Undo changes in this file.

Reply:
These were removed in #6803 [here](https://github.com/oppia/oppia/commit/8d7533384651c2f49f1bc04e64b86c8e42c585e9#diff-32607347f8126e6534ebc7ebaec4853d).